### DISCUSSION TITLE
Where can I see download status?

### DISCUSSION THREAD

**Discussion by vorob1** [section83/MacYTDL#76] [General]
I've started download process and status window appeared in upper right corner. I pressed close now I I've got not idea where to look for it :)

**Comment by section83** [comment]
Hello, thank you for making contact.

There is no way to see download progress after closing the progress and status window (I call it the "Monitor" dialog). This is because of the kind of dialog I've used which can't be hidden or minimised and when closed, the process ceases.  The Monitor dialog is a separate process and closing it does not affect the download.

Of course, you can see the download in progress by viewing your download folder.  By default, yt-dlp saves files ending in ".part" which will change a lot as the download progresses. MacYTDL issues an alert when the download is finished.  The alert stays visible for 10 seconds.

You can also check whether a download is still underway by opening "Activity Monitor".  There is a yt-dlp process for each download.  You can quit a yt-dlp process to stop the download.

The Monitor dialog in the next version of MacYTDL will take up less space on the screen (I've removed unnecessary text).

I've wanted to improve the Monitor for a long time and am hoping to do something later this year.

Cheers.

Garry

**Comment by vorob1** [comment]
No problem, thank you for your app!
